src/hash.cr

```diff
@@ -1239,7 +1239,7 @@ class Hash(K, V)
     @size == 0
   end
 
-  # Calls the given block for each key-value pair and passes in the key and the value.
+  # Calls the given block for each key-value pair and passes in the key and the value as `each(& : Tuple(K, V) ->) : Nil`
```

Comment:
This sounds strange to me, what do you want to convey here?

Reply:
Hopefully avoided it by moving signature to the method itself now, thanks!